.github/workflows/ci.yml

```diff
@@ -51,6 +48,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 50
```

Comment:
Does this need to be 50 instead of 0 or 2?

Reply:
This need to be enough to fetch branch base. 0 means all repo, which is safe but can slowdown the build when the repo grows. I think this is safe to expect PR to be less than 50 commits. If not, the build fails because the git diff command will fail.

Reply:
This need to be enough to fetch branch base. 0 means all repo, which is safe but can slowdown the build when the repo grows. I think this is safe to expect PR to be less than 50 commits. If not, the build fails because the git diff command will fail.

Reply:
I see. 👍